src/reformatters/noaa/hrrr/forecast_48_hour/template_config.py

```diff
@@ -127,55 +144,86 @@ def get_init_time_coordinates(
+        name="x",
+        encoding=Encoding(
+            dtype="int64",
+            fill_value=np.nan,
+            compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
+            chunks=len(_dim_coords["x"]),
+            shards=len(_dim_coords["x"]),
+        ),
+        attrs=CoordinateAttrs(
+            units="unitless",
+            statistics_approximate=StatisticsApproximate(
+                min=_dim_coords["x"].min(),
+                max=_dim_coords["x"].max(),
+            ),
+        ),
+    ),
+    Coordinate(
+        name="y",
+        encoding=Encoding(
+            dtype="int64",
+            fill_value=np.nan,
+            compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
+            chunks=len(_dim_coords["y"]),
+            shards=len(_dim_coords["y"]),
+        ),
+        attrs=CoordinateAttrs(
+            units="unitless",
+            statistics_approximate=StatisticsApproximate(
+                min=_dim_coords["y"].min(),
+                max=_dim_coords["y"].max(),
+            ),
+        ),
+    ),
+    Coordinate(
+        name="latitude",
+        encoding=Encoding(
+            dtype="float64",
+            fill_value=np.nan,
+            compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
+            chunks=(len(_dim_coords["y"]), len(_dim_coords["x"])),
+            shards=(len(_dim_coords["y"]), len(_dim_coords["x"])),
+        ),
+        attrs=CoordinateAttrs(
+            units="degrees_north",
+            # TODO: How to set these min/max values?
+            statistics_approximate=StatisticsApproximate(
+                min=_dim_coords["y"].min(),
+                max=_dim_coords["y"].max(),
+            ),
+        ),
+    ),
+    Coordinate(
+        name="longitude",
+        encoding=Encoding(
+            dtype="float64",
+            fill_value=np.nan,
+            compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
+            chunks=(len(_dim_coords["y"]), len(_dim_coords["x"])),
+            shards=(len(_dim_coords["y"]), len(_dim_coords["x"])),
+        ),
+        attrs=CoordinateAttrs(
+            units="degrees_east",
+            # TODO: How to set these min/max values?
+            statistics_approximate=StatisticsApproximate(
+                min=_dim_coords["x"].min(),
+                max=_dim_coords["x"].max(),
+            ),
+        ),
+    ),
     Coordinate(
         name="valid_time",
         encoding=Encoding(
             dtype="int64",
             fill_value=0,
-            # TODO
-            # compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
+            compressors=[BLOSC_8BYTE_ZSTD_LEVEL3_SHUFFLE],
             calendar="proleptic_gregorian",
             units="seconds since 1970-01-01 00:00:00",
-            chunks=(INIT_TIME_COORDINATE_CHUNK_SIZE, len(_dim_coords["lead_time"])),
-            shards=(INIT_TIME_COORDINATE_CHUNK_SIZE, len(_dim_coords["lead_time"])),
+            chunks=INIT_TIME_COORDINATE_CHUNK_SIZE,
+            shards=INIT_TIME_COORDINATE_CHUNK_SIZE,
```

Comment:
I have a feeling this is the source of the change. 

Does `valid_time` have both of these dimensions? I suppose it does, now that I'm thinking of it...